### ISSUE TITLE
📎 Reduce command suggestion console noise

### ISSUE THREAD

**Issue by sebmck** [rome/tools#1362]
```
 internal/path/index.ts lint/pendingFixes  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Pending formatting and safe fixes

    1087 1087 │   }
    1088 1088 │
    1089      │ - export·const·UNKNOWN_PATH·=·createUIDPath("unknown");
         1089 │ + export·const·UNKNOWN_PATH·=·createUIDPath("unknown");
         1090 │ +

  ℹ Only difference is leading and trailing whitespace

  ℹ To apply fixes and formatting run
  $ rome check internal/path/index.ts --apply


━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

ℹ Fixes available. To apply safe fixes and formatting run
$ rome check --apply
ℹ To choose fix suggestions run
$ rome check --review

✖ Found 6 problems
```

We suggest too many commands. Not sure what the right balance is.

**Comment by ematipico** [comment]
It would be great to receive some feedback from the adopter!

**Comment by jer3m01** [comment]
I'd remove the per diagnostic command suggestion and just keep the 2 at the end (and maybe change the wording/add spacing).

```
 internal/path/index.ts lint/pendingFixes  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Pending formatting and safe fixes

    1087 1087 │   }
    1088 1088 │
    1089      │ - export·const·UNKNOWN_PATH·=·createUIDPath("unknown");
         1089 │ + export·const·UNKNOWN_PATH·=·createUIDPath("unknown");
         1090 │ +

  ℹ Only difference is leading and trailing whitespace


━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

ℹ Fixes available. To apply safe fixes and formatting run
$ rome check --apply

ℹ To interactively choose fix suggestions run
$ rome check --review


✖ Found 6 problems
```

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
